btree/btree.c

```diff
@@ -8583,7 +8603,10 @@ M0_INTERNAL int64_t m0_btree_lrulist_purge_check(enum m0_btree_purge_user user,
 	 * or size whichever is higher.
 	 */
```

Comment:
In the above if-condition - if (lru_space_used < lru_space_wm_high)
you will need to check if the lru_slow_purge is ON and accordingly purge the nodes till the lru_space_used falls below target_wm.

Reply:
https://github.com/Seagate/cortx-motr/pull/1546#discussion_r831021444